packages/adapter-node/src/server_base.js

```diff
@@ -0,0 +1,56 @@
+import compression from 'compression';
+import fs from 'fs';
+import polka from 'polka';
+import { dirname, join } from 'path';
+import sirv from 'sirv';
+import { URL, fileURLToPath } from 'url';
+// eslint-disable-next-line import/no-unresolved
+import { get_body } from '@sveltejs/kit/http';
+// App is a dynamic file built from the application layer.
+/*eslint import/no-unresolved: [2, { ignore: ['\.\/app\.js$'] }]*/
+import * as app from './app.js';
+
+const __dirname = dirname(fileURLToPath(import.meta.url));
+
+const mutable = (dir) =>
+	sirv(dir, {
+		etag: true,
+		maxAge: 0
+	});
+
+const noop_handler = (_req, _res, next) => next();
+
+const prerendered_handler = fs.existsSync('prerendered') ? mutable('prerendered') : noop_handler;
```

Comment:
perhaps this stuff should live inside `createServer`? it we're moving in that direction it probably doesn't make sense to create the middleware on import